### ISSUE TITLE
"Movement" option?

### ISSUE THREAD

**Issue by goldencow2** [18xx-maker/18xx-maker#342]
Some games have a "movement" section defined inside the "stock" section that would display the ways share price can move, but none of the examples seem to actually display in the app.

**Comment by kelsin** [comment]
Yeah, the old display wasn't very flexible and I'm looking into how I want to do this. Thinking of something similar to the way that 1846 displays it on the map, and allowing it to be put on the map or the market (like the round tracker and play charts now). Open to suggestions (mock ups etc) or PRs.
